LICENSE.md

```diff
@@ -1,66 +1,117 @@
-11.13. Controlling Language. This Agreement is prepared an executed in the English language only, which language shall be controlling in all respects. Any translations of this Agreement into any other language are for reference only and shall have no legal or other effect. Any notice which is required or permitted to be given by one party to the other under this Agreement will be in the English language and will be in writing. All proceedings related to this Agreement shall be conducted in the English language.
\ No newline at end of file
+# BigDesign UI Kit
+Thank you for your interest in BigDesign.  BigDesign is an open source project subject to the below open source licenses for the different UI components of BigDesign. **BY DOWNLOADING, INSTALLING, OR OTHERWISE USING THE BIGDESIGN PROJECT YOU AGREE TO ABIDE BY THIS LICENSE**. IF YOU DO NOT AGREE WITH THE TERMS AND CONDITIONS OF THIS LICENSE YOU ARE NOT AUTHORIZED TO USE BIGDESIGN.
+
+### Disclaimer
+BigCommerce, Inc. is and remains the owner of all right, title, and interest in its e-commerce platform and other products and services, including all rights in the BigCommerce® mark and other trademarks and logos of BigCommerce. Further, your participation in the BigDesign program may be subject to developer terms, API terms, or other BigCommerce terms relevant to your design of offerings that interface with the BigCommerce platform. BigCommerce reserves all rights not expressly granted hereunder.
+
+## BigDesign Software Code
+**Applies to**: All CSS and JS files in at https://github.com/bigcommerce/big-design.
+
+**License**:  MIT License (http://opensource.org/licenses/mit-license.html)
+
+**License Text**:
+
+Copyright © 2021 BigCommerce, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+## BigDesign UI Elements
+**Applies to**:  All elements of the UI Kit at https://github.com/bigcommerce/big-design that are not software code, including documentation, icons, style sets, typography, and buttons.
```

Comment:
Reading this again, I'm not sure this is actually a problem, since this is _excluding_ style sets (which we don't have) from consideration (which would make them fall under the MIT license part). I think this is fine?

Reply:
CSS definitely counts as code, SVG is a bit fuzzier since it's not really code in an _executable_ context, it's an encoding of an image (which all images technically are?)

The provided template left the URL blank so I can just add the Figma URL here too so it's clear that it extends to that content?